v3/src/components/graph/adornments/adornments.tsx

```diff
@@ -36,7 +36,7 @@ export const Adornments = observer(function Adornments() {
       layout.setDesiredExtent("banners", bannersHeight)
       }, { name: "Graph.handleAdornmentBannerCountChange" }, graphModel
     )
-  }, [dataConfig, graphModel, layout])
+  }, [dataConfig?.showMeasuresForSelection, graphModel, layout])
```

Comment:
I don't think you want/need this change. This says that when `showMeasuresForSelection` changes, the autorun should be disposed and reinstalled. `showMeasuresForSelection` is already accessed inside the autorun, so the autorun will be triggered when it changes without needing to dispose it and reinstall it.
```suggestion
  }, [dataConfig, graphModel, layout])
```